### PR TITLE
feat: Rangescalar and other transformer implementations now pass exceptions from sklearn

### LINKED ISSUE #326: `RangeScaler` should not pass exception from scikit-learn when `fit`ting on empty table
### Is your feature request related to a problem?

See title.

### Desired solution

* Should throw new, user-friendly exception.
* Verify that exception is thrown.

Should also check if any the other `Transformer` implementations pass on exceptions from scikit-learn.

### PR DESCRIPTION
**Pull request by jxnior01**
Closes #326 

### Summary of Changes

* Rangescalar and other transformer implementations now pass exceptions from sk-learn when fitting on an empty table

### CODE REVIEW

**Comment by zzril** [comment]
We *don't* want the exceptions from scikit-learn to be passed on to the users. Instead, we want understandable, user-friendly exceptions.

The original issue was about one case where such an exception from sklearn *was* (unintentionally) passed through.

I'll need to check if that one has already been fixed by now. If so, I'll close the issue.

**Comment by zzril** [comment]
Yeah, I guess the title was somewhat misleading. I thought I had given it a "bug" label, in which case the old title would have made for sense.
Anyway, changed the title now. Sorry for the confusion.
